### DISCUSSION TITLE
Feature Proposal

### DISCUSSION THREAD

**Discussion by SitiSchu** [dooit-org/dooit#163] [Ideas]
I was missing some features in dooit so I went ahead and added, now the questions is which ones I should PR:
1. Shortcut to recursively expand node trees, when having multiple nested todos it can be annoying to have to open them all manually
2. Config option to start dooit with the workspace or todo tree fully expanded, configurable seperately for workspace and todos
3. Config option to define the initial urgency for new todos, useful to have todos start at urgency 2 for example, and then have the ability to mark a todo as low priority by setting it's urgency to 1
4. Config option to configure urgency colors, primarily useful together with 3. as this allows setting urgency to grey and urgency 2 to green for example.
5. Move `main()` out of `__init__` and into `__main__`, this will allow dooit to be executed using `python -m dooit` 

If any of those features are wanted I can either PR them all together as one or do a seperate PR for each

**Comment by kraanzu** [comment]
Great suggestions!

> Config option to configure urgency colors, primarily useful together with 3. as this allows setting urgency to grey and urgency 2 to green for example.

Great for now, they can just provide a hex code for the color. In the future, I was thinking about introducing themes that will reduce dependency on config files as well as provide a balanced look as well :)

**Reply by SitiSchu** [reply]
I'll go ahead an PR them then, I'll combine them into one PR so it's easier to merge ^^
